### PR TITLE
c# v10

### PR DESCRIPTION
**Pull request by shargon**
Close https://github.com/neo-project/neo-devpack-dotnet/issues/711

### CODE REVIEW

**Review by Liaojinghui — APPROVED** [review]

**Comment by erikzhang** [comment]
There is a bug in Microsoft.CodeAnalysis.CSharp 4.0.1, which prevents us from eliminating those warnings. Wait for 4.1.0.

**Comment by shargon** [comment]
> There is a bug in Microsoft.CodeAnalysis.CSharp 4.0.1, which prevents us from eliminating those warnings. Wait for 4.1.0.

@erikzhang It's released, please review it

**Review by erikzhang — APPROVED** [review]

File: src/Neo.Compiler.CSharp/Neo.Compiler.CSharp.csproj [hunk 1]
```diff
@@ -21,7 +21,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.1.0" />
     <PackageReference Include="Neo" Version="3.1.0" />
-    <PackageReference Include="System.CommandLine" Version="2.0.0-beta1.21308.1" />
+    <PackageReference Include="System.CommandLine" Version="2.0.0-beta3.22114.1" />
```
Comment (shargon) [inline src/Neo.Compiler.CSharp/Neo.Compiler.CSharp.csproj:24]: Revert this change?

Comment (erikzhang) [reply src/Neo.Compiler.CSharp/Neo.Compiler.CSharp.csproj:24]: Fixed.

**Review by erikzhang — APPROVED** [review]
